Sources/ITwinMobile/ITMActionSheet.swift

```diff
@@ -38,6 +38,8 @@ final public class ITMActionSheet: ITMNativeUIComponent {
                 // When an action is selected, this gets called before the action's handler.
                 // By running async in the main event queue, we delay processing this until
                 // after the handler has had a chance to execute.
+                // NOTE: Task { @MainActor won't work below, because we're already on the UI thread when
+                // we get here; using DispatchQueue.main.async forces the code inside to run later.
                 DispatchQueue.main.async {
```

Comment:
Can we use MainActor.run instead? Not sure if it's desirable but thought I'd ask.

Reply:
I did test this, and it seems to work. I thought about changing it to use that, but decided not, since I can't be sure that it is guaranteed to work forever. (In other words, at some point in the future it might decided not to produce a delay.)